examples/mass_spring_3d.py

```diff
@@ -79,31 +85,49 @@ def init_display():
         i.x -= 1
         d = i.dot(tl.vec(N, 1))
         i.y -= 1
-        model.faces[a * 4 + 0] = [a, c, b]
-        model.faces[a * 4 + 1] = [a, d, c]
-        model.faces[a * 4 + 2] = [a, b, c]
-        model.faces[a * 4 + 3] = [a, c, d]
+        for _ in ti.static(range(3)):
+            for __ in ti.static(range(3)):
+                model.faces[a * 4 + 0][_, __] = [a, c, b][_]
+                model.faces[a * 4 + 1][_, __] = [a, d, c][_]
+        for _ in ti.static(range(3)):
+            for __ in ti.static(range(2)):
+                model.faces[a * 4 + 2][_, __] = [a, b, c][_]
+                model.faces[a * 4 + 3][_, __] = [a, c, d][_]
```

Comment:
nit: please avoid using `_` and `__` as loop variables. It can be slightly hard to read. Perhaps `p, q`, `u, v`, `r, s` are better options?